### DISCUSSION TITLE
AuthProvider: Clerk

### DISCUSSION THREAD

**Discussion by Nosferatu31** [refinedev/refine#5488] [Integration request] [answered]
I searched for it but couldn't find it so this is a request (I guess).
I am referring to: https://clerk.com/

**Comment by BatuhanW** [comment, marked as answer]
Hello @Nosferatu31, we are open to contributions from the community. We'd be more than happy to review PRs, and/or add community libraries to our community packages page in our documentation.

**Reply by Nosferatu31** [reply]
Thank you for the reply Batuhan, if I manage to come up with a sound workaround, I'll contribute with it :smile:

**Reply by oldo** [reply]
@Nosferatu31 how did you go? I am looking at using Refine but would need to use Clerk. Did you have success with setting up a successful integration and was it fairly straight forward?
Thanks.
